archivematica/startTransfer.py

```diff
@@ -1,10 +1,7 @@
 #!/usr/bin/env python
 
-import base64
-import json
+import base64, json, requests, time
```

Comment:
The way you had this before was right, according to PEP 8: https://www.python.org/dev/peps/pep-0008/#imports